requirements.in

```diff
@@ -14,6 +14,4 @@ pytest-mock
 structlog
 
 opentracing>=2.4.0
-confluent-kafka>=1.0.0,<=1.9.0
-avro-python3>=1.8.2,<=1.10.2
-fastavro>=0.18.0,<=1.5.1
+confluent-kafka[avro,schema-registry]>=1.0<=1.9
```

Comment:
`avro` will install `avro-python` and `fastavro`, and `schema-registry` will install `requests`